app/classes/Transvision/ShowResults.php

```diff
@@ -403,6 +418,13 @@ public static function resultsTable($search_object, $search_results, $page)
             // Replace / and : in the key name and use it as an anchor name
             $anchor_name = str_replace(['/', ':'], '_', $key);
 
+            $string_id = md5($target_string);
+            $regular_string_id = 'string_' . $string_id;
+            if ($locale2 == 'sr' && !$extra_locale) {
+                $transliterated_string = self::getTransliteratedString($target_string, 'sr-Cyrl');
```

Comment:
we actually need to do this call right after we define $target_string, and before we do all the transforms  (highlight, markString, htmlspecialchars…), otherwise tags are displayed in the transliterated string. We will also need to apply the same transforms on our transliterated string.
I think everything is here https://github.com/mozfr/transvision/blob/master/app/classes/Transvision/ShowResults.php#L318-335